util/deploy.sh

```diff
@@ -37,6 +37,7 @@ done
 if [[ "${APP_PATH}" == ""  ]]; then fatal "app path not specified."; fi
 case "${APP_PATH}" in
   "webapp/web" | \
+  "webapp/web/app.staging.yaml" | \
```

Comment:
My understanding in deployment in travis is rather vague but it looks like `APP_PATH` is changed as a result. Would it affect places like [travis-deploy-staging.sh](https://github.com/web-platform-tests/wpt.fyi/blob/master/util/travis-deploy-staging.sh#L24)?

Reply:
Good catch! Fixed.